Packs/GroupIB_ThreatIntelligenceAttribution/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,620 @@
+
```

Comment:
As some fields were deleted, please add Breaking Changes release notes (using the `bc` flag in `demisto-sdk update-release-notes`)

Reply:
Hi @idovandijk , can you please advise on this point?